lib/enhance-assert.js

```diff
@@ -12,23 +13,40 @@ module.exports.PATTERNS = [
 	't.regexTest(regex, contents, [message])'
 ];
 
-function enhanceAssert(assert) {
-	var empower = require('empower');
-	var powerAssertFormatter = require('power-assert-formatter');
-	var powerAssertRenderers = require('power-assert-renderers');
+module.exports.NON_ENHANCED_PATTERNS = [
+	't.pass([message])',
+	't.fail([message])',
+	't.throws(fn, [message])',
+	't.doesNotThrow(fn, [message])',
+	't.ifError(error, [message])'
+];
 
-	empower(assert,
-		powerAssertFormatter({
-			renderers: [
-				powerAssertRenderers.AssertionRenderer,
-				powerAssertRenderers.SuccinctRenderer
-			]
-		}),
+function enhanceAssert(opts) {
+	var empower = require('@jamestalmage/empower-core');
+	var enhanced = empower(
+		opts.assert,
 		{
-			destructive: true,
-			modifyMessageOnRethrow: true,
-			saveContextOnRethrow: false,
-			patterns: module.exports.PATTERNS
+			destructive: false,
+			onError: opts.onError,
+			onSuccess: opts.onSuccess,
+			patterns: module.exports.PATTERNS,
+			additionalMethods: module.exports.NON_ENHANCED_PATTERNS
```

Comment:
It's new - and it will be end up being called `wrapOnlyPatterns`. This is for functions that it does not make sense to provide the big graph for (i.e., any use of `t.throws` is likely to contain an inline function and be way to complicated for a power-assert renderer to do a good job with). By adding this though, we still wrap those and give you the assertion `events`, so you don't have to handle "enhanced" and "non-enhanced" separately. You get events for both.